testing/test/extensions/public/AzureMLKubernetes.Tests.ps1

```diff
@@ -58,11 +58,8 @@ Describe 'AzureML Kubernetes Testing' {
     It "Deletes the extension from the cluster with inference enabled" {
```

Comment:
Can you add an update scenario to the test cases so that we are validating the update scenario works as expected?

Reply:
Will do